compute/instances.go

```diff
@@ -28,8 +29,8 @@ const (
 // values are embedded within a Instance's Tags attribute, however they are
 // exposed to the caller as their native types.
 type InstanceCNS struct {
-	Disable    *bool
-	ReversePTR *string
+	Disable    bool
+	ReversePTR string
```

Comment:
We should be fine changing these semantics considering this struct was renamed under the new API.